CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ For older changes see the [archived Singularity change log](https://github.com/a
 
 ### Changed defaults / behaviours
 
+- Make `PS1` envrionment variable changeable via `%environment` section on
+  definition file that used to be only changeable via `APPTAINERENV_PS1`
+  outside of container. This makes container's prompt customizable.
+  The line `export PS1="YOUR_PS1_VALUE>"` on `%environment` section gives
+  you prompt `YOUR_PS1_VALUE>` inside container. This change is only affect
+  to newly built containers, but not affect to old containers.
```

Comment:
I suggest deleting these two sentences.  They both have problematic english which could be corrected, but I think the first one is redundant.  Even though I had asked for the second one, I can see now that the fact is obvious since it only applies to `%environment` which applies to new containers.  When I asked for it I didn't realize that setting `APPTAINERENV_PS1` already worked before this patch.

Reply:
Thank you @DrDaveD! I reflected your feedback. I deleted L15 to L17.